crates/pixi_build_frontend/src/protocols/conda_build/protocol.rs

```diff
@@ -0,0 +1,278 @@
+        } else {
+            conda_render_executable
+        };
+
+        let conda_render_tool = tool.with_executable(conda_render_executable);
+
+        // TODO: Properly pass channels
+        // TODO: Setup --exclusive-config-files
+
+        let channels = request
+            .channel_base_urls
+            .iter()
+            .flatten()
+            .flat_map(|url| ["--channel", url.as_str()]);
+
+        let output = conda_render_tool
+            .command()
+            // .arg("--verbose")
+            // This is currently apparently broken in conda-build..
+            // .arg("--use-channeldata")
+            .arg("--override-channels")
+            .args(channels)
+            .arg(&self.recipe_dir)
+            .stderr(std::process::Stdio::inherit())
+            .stdout(std::process::Stdio::piped())
+            .output()
+            .into_diagnostic()
+            .context("failed to spawn conda-render executable")?;
+
+        // Try to parse the contents of the output.
+        let stdout = String::from_utf8(output.stdout)
+            .into_diagnostic()
+            .context("failed to convert the output of conda-render to a valid utf-8 string")?;
+
+        // Fail if the process did not exit successfully.
+        if !output.status.success() {
+            miette::bail!(
+                "conda-render returned with a non-zero exit code:\n{}",
+                stdout
+            );
+        }
+
+        // Parse the output of conda-render.
+        let rendered_recipes = extract_rendered_recipes(&stdout)?;
+
+        let metadata = CondaMetadataResult {
+            packages: rendered_recipes
+                .into_iter()
+                .map(|(recipe, meta_yaml)| {
+                    convert_conda_render_output(recipe).with_context(|| {
+                        format!(
+                            "failed to extract metadata from conda-render output:\n{}",
+                            meta_yaml
+                        )
+                    })
+                })
+                .collect::<miette::Result<_>>()?,
+            input_globs: None,
+        };
+
+        Ok(metadata)
+    }
+}
+
+/// Given output from `conda-render`, parse it into one or more
+/// [`CondaRenderRecipe`]s.
+fn extract_rendered_recipes(
+    rendered_recipe: &str,
+) -> miette::Result<Vec<(CondaRenderRecipe, &str)>> {
+    static OUTPUT_REGEX: OnceLock<Regex> = OnceLock::new();
+    let output_regex = OUTPUT_REGEX.get_or_init(|| {
+        Regex::new(r#"(?sR)Hash contents:\r?\n-{14}\r?\n(.+?)-{10}\r?\nmeta.yaml:\r?\n-{10}\r?\n(.+?)(?:-{14}|$)"#)
+            .unwrap()
+    });
+
+    let mut iter = output_regex.captures_iter(rendered_recipe).peekable();
+    if iter.peek().is_none() {
+        miette::bail!(
+            "could not find metadata in conda-render output:\n{}",
+            rendered_recipe
+        )
+    }
+
+    iter.map(|captures| {
+        let hash = captures.get(1).unwrap().as_str().trim();
```

Comment:
Are these unwraps real unwraps or should they become an error?

Reply:
I think they will even dissapear because we plan to move conda-build into a separate backend. 
even more, I think currently conda-build is not in that good working state because you cannot build it